User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+<!--- Explanation of what Trust Levels, how they are calculated, and how to earn then -->
+
+# Trust Level Overview
+
+Because Codidact is community run, it doesn't have owners who dictate behavior. Instead, users who consistently make solid contributions through posting and other activity on a community gradually obtain more privileges and access to certain moderation tools based on a calculated "Trust Level." You can see current Trust Level on any user's profile. 
+
+Some communities may have a higher bar to obtain trust levels than others, or restrict certain categories based on trust level. The default settings are listed below, but check out your site's FAQ to see if they have any custom requirements. 
+
+# Trust Levels and associated permissions
+
+There are five trust levels with increasing access and levels of responsibility. The table below shows the typical community settings for each. 
+
+|                                       | TL0 - Newcomer                                                                                 | TL1 - New Member                                                                                                 | TL2 - Standard Member                                             | TL3 - Power Member                                                                                                | TL4 - Deputy                                                                              | TL5 - Moderator                                                |
```

Comment:
I'm going to assume these are the same as in the spreadsheet and rely on that instead of reading the linewrap here. 

We talked about this in chat, but for locks, instead of temp & perm lock, consider just one line for lock, with TL4 being "temporary" and TL5 being "permanent".

Ultimately we'll probably need to link other help pages for several of these ("what's a lock?"), but we can't do that until we have them.  But let's keep this terse as you have now instead of trying to pack all that info into the table here.